tests/MongoDB.EntityFrameworkCore.SpecificationTests/Query/NorthwindQueryMongoFixture.cs

```diff
@@ -46,6 +46,44 @@ public TestMqlLoggerFactory TestMqlLoggerFactory
     protected override bool ShouldLogCategory(string logCategory)
         => logCategory == DbLoggerCategory.Query.Name;
 
+#if !EF9
+    protected override void Seed(NorthwindContext context)
```

Comment:
Why is this one only foe EF8? The EnsureCreated is available on EF8 and surely needs it too?

Reply:
(A bunch of stuff switched to async in the EF9 tests because Cosmos doesn't have any sync APIs.)